charts/jenkins/values.yaml

```diff
@@ -651,9 +651,11 @@ agent:
     requests:
       cpu: "512m"
       memory: "512Mi"
+      ephemeralStorage: ""
```

Comment:
```suggestion
      # ephemeralStorage:
```

Reply:
Sure, the "" value is null is valid to pass in, and the conditional should stop it from being put into the cluster/plugin unless you actually insert a value.

I don't really mind one way or the other.